string  posture_name
---
meka_posture_recorder/PostureRecordErrorCodes error_code
